modules/kubernetes/aks-core/modules.tf

```diff
@@ -272,10 +272,12 @@ module "datadog" {
 
   source = "../../kubernetes/datadog"
 
-  location     = var.location_short
-  environment  = var.environment
-  datadog_site = var.datadog_config.datadog_site
-  api_key      = var.datadog_config.api_key
+  location                 = var.location_short
+  environment              = var.environment
+  datadog_site             = var.datadog_config.datadog_site
+  api_key                  = var.datadog_config.api_key
+  app_key                  = var.datadog_config.app_key
+  container_filter_include = var.datadog_config.container_filter_include
```

Comment:
I would not make this configurable if there is no need to.

Reply:
Fixed